src/Core/Components/AppBar/FluentAppBar.razor

```diff
@@ -23,7 +23,7 @@
                                  HorizontalAlignment="HorizontalAlignment.Center"
                                  VerticalAlignment="VerticalAlignment.Center"
                                  VerticalGap="0"
-                                 Style="width: 100%; --appbar-active-indicator-width: calc(var(--design-unit)* 0.5px); height: 56px;">
+                                 Style="--appbar-active-indicator-width: calc(var(--design-unit)* 0.5px); width: 56px; height: 56px;">
```

Comment:
Why not to use `var(--appbar-item-height, 56px)` for width and height (like done in `FluentAppBar.razor.css` ?

Reply:
I'm creating a new CSS var for this: --appbar-item-size